user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -221,7 +221,9 @@ Within the dashboard or app URL, the following data objects can be specified:
 
     `?query columns=3af9e5a7-91fc-4333-94c0-e39a59f0d900%1e23ea428c-d52c-4041-8fd9-76ce3f436a6d_Number%1erange%1e5%1f10%1ffalse%1ftrue`
 
-- *query rows*: Supported from DataMiner 10.3.0 [CU12]/10.4.3 onwards<!-- RN 39369 -->. An array of query rows. Each row must be in the following format: `VERSION\u001FCOLUMNS\u001ECELLS\u001EKEYS`.
+- *tables*: Supported from DataMiner 10.3.0 [CU12]/10.4.3 onwards<!-- RN 39369 -->.  An array of query rows. Prior to DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12, this object is called "query rows" instead.
```

Comment:
```suggestion
- *query rows*: Supported from DataMiner 10.3.0 [CU12]/10.4.3 onwards<!-- RN 39369 -->.  An array of query rows.
```

So, if I'm correct, these objects should be reverted back to "strings" and "query rows", @Robin-Debel?

Reply:
I believe this might be better yes, to use it into interpolation however you will need to use the new terms: {URL."Query parameters".Texts.Value}

Reply:
I've added two notes to the "Dynamically referencing data in text" page. However, I've included this update in the "Feeds" draft pull request to prevent any merge conflicts, as I've completely restructured this page. Could you double-check [this commit](https://github.com/SkylineCommunications/dataminer-docs/pull/3880/commits/510f8f61ece12b2307a30abb1acfc3ab0fae6efb), @Robin-Debel?

Reply:
Looks ok to me